site/specs/numbers.json

```diff
@@ -849,220 +846,204 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves the phone numbers according to the input parameters</p>\n",
-        "operationId": "GET /accounts/{accountId}/availableNumbers",
+        "description": "Retrieves the phone numbers according to the input parameters.",
+        "operationId": "GetAvailableTns",
+        "summary": "Retrieve available tns",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
             "in": "path",
             "required": true,
+            "style": "simple",
+            "explode": false,
+            "description": "User's account ID",
+            "example": "900000",
             "schema": {
-              "type": "string",
-              "default": null
-            },
-            "example": null
+              "type": "string"
+            }
           },
           {
             "name": "quantity",
-            "description": "<p>The desired quantity of requested numbers. Values range from 1-5000.</p>\n",
+            "description": "The desired quantity of requested numbers.",
             "in": "query",
             "required": false,
             "schema": {
               "type": "integer",
+              "minimum": 1,
+              "maximum": 5000,
               "default": 5000
             },
             "example": 30
           },
           {
             "name": "enableTNDetail",
-            "description": "<p>If set to true, a list of details associated with the telephone number (rate center, abbreviation, city, state, and LATA) will be displayed along with the TN.</p>\n",
+            "description": "If set to true, a list of details associated with the telephone number (rate center, abbreviation, city, state, and LATA) will be displayed along with the TN.",
             "in": "query",
             "required": false,
             "schema": {
               "type": "boolean",
               "default": false
-            },
-            "example": null
+            }
           },
           {
             "name": "LCA",
-            "description": "<p>Values are true or false.</p>\n",
+            "description": "<p>Values are true or false.</p>",
```

Comment:
You can probably do a regex find and replace to remove all `<p>` tags in descriptions

Reply:
It would only be complicated for descriptions with multiple `<p>` tags.  It's not a huge deal to have these in here I guess, it just makes it less verbose.